### DISCUSSION TITLE
Detecting if a given sound effect is playing?

### DISCUSSION THREAD

**Discussion by mooncalf** [nesbox/TIC-80#1861] [General]
Is there any way to tell if a particular sound effect (by ID) is currently playing?

**Comment by nesbox** [comment]
Unfortunately, there is no way.

**Reply by mooncalf** [reply]
I wrote my own littler handler, not a problem! Thanks for your work, loving it. Just finished my first game, will post.

**Reply by spooneystone** [reply]
care to share how you handled it? could be handy for someone
